framework/src/modules/chain/chain.js

```diff
@@ -52,7 +53,6 @@ const config = {
 		loader: './modules/loader.js',
 		multisignatures: './modules/multisignatures.js',
 		peers: './modules/peers.js',
-		system: './modules/system.js',
 		signatures: './modules/signatures.js',
 		transactions: './modules/transactions.js',
 		transport: './modules/transport.js',
```

Comment:
This is not directly relevant to this PR, but I find the property name `modules` confusing here in the context of lisk-modular.

It seems that we should make a clear distinction between modules and components within our code. Is this something that is planned to be changed later?

Reply:
Please, raise this concern to @nazarhussain.